### DISCUSSION TITLE
ebonheart pact guide

### DISCUSSION THREAD

**Discussion by nvrkc** [Gator7778/CGESO#14] [General]
only includes bleakrock isle. thought it would include 1-50 main quest ?

and when you complete bleakrock, it tries to load stonefalls MSQ but it doesnt exist

**Comment by Gator7778** [comment]
if your talking about Main Quest Only its because i havent touched that yet i been redoing all these guides. And i plan on redoing that from scratch im currently working on the full dlc guides. that was done solely by Zygor and hasnt been touched since we took it over but i have to plans to run it all the way through Deadlands
